packages/forms-web-app/src/lib/client-side/google-tag-manager.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-env browser */
+
+function applyConsent(consent) {
+  window.dataLayer = window.dataLayer || [];
+
+  function gtag() {
+    // eslint-disable-next-line no-undef, prefer-rest-params
+    dataLayer.push(arguments);
+  }
+
+  if (consent === 'denied' || consent === 'granted') {
```

Comment:
These unit tests look good but the coverage is failing on this conditional due to the else path not being tested.

I just looked into it a bit and this means it either needs 

1. a test to test the scenario where `consent` is set to something other than `denied` or `granted`
2. tell istanbul to ignore the else condition with `/* istanbul ignore else  */` on the line before the conditional

Happy either way because of the way this code is currently written a scenario where `consent` doesn't equal one of these values isn't possible.